djangocms_references/helpers.py

```diff
@@ -257,7 +243,14 @@ def apply_additional_modifiers(queryset):
     return queryset
 
 
-def get_all_reference_objects(content, draft_and_published=False):
+def apply_filters(queryset, model, state_selected):
+    if get_versionable_for_content(model):
+        queryset = queryset.filter(versions__state__in=([state_selected]))
+        return queryset
+    return queryset
+
+
+def get_all_reference_objects(content, state_selected=None, model=None):
```

Comment:
The params have changed but the documentation within the function has been neglected. This package has been very well documented, it's how code should be documented and something that we should be doing with any new code.